README.md

```diff
@@ -21,13 +21,15 @@ A state controller with its own debugger
 ## Try out Cerebral 2
 To try out Cerebral 2 you can install any package with:
 
-`npm install {cerebralPackage}@next --save --save-exact`
+`npm install {cerebralPackage}@beta --save`
+
+For packages which are scoped with `@cerebral/` `@beta` can be omitted.
```

Comment:
@Guria Why you want to keep beta tag? There are no Cerebral 1 packages with this name and after release the beta tag will be removed.